app/src/main/java/com/github/factotum_sdp/factotum/ui/display/DisplayFragment.kt

```diff
@@ -6,75 +6,81 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.activity.OnBackPressedCallback
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
 import androidx.fragment.app.viewModels
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
-import androidx.lifecycle.ViewModelProvider
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.github.factotum_sdp.factotum.R
 import com.github.factotum_sdp.factotum.UserViewModel
 import com.github.factotum_sdp.factotum.databinding.FragmentDisplayBinding
-import com.github.factotum_sdp.factotum.ui.display.utils.PhotoAdapter
+import com.github.factotum_sdp.factotum.models.Role
+import com.github.factotum_sdp.factotum.ui.display.boss.BossDisplayViewModel
+import com.github.factotum_sdp.factotum.ui.display.boss.BossFolderAdapter
+import com.github.factotum_sdp.factotum.ui.display.client.ClientDisplayViewModel
+import com.github.factotum_sdp.factotum.ui.display.client.ClientDisplayViewModelFactory
+import com.github.factotum_sdp.factotum.ui.display.client.ClientPhotoAdapter
 import com.google.firebase.storage.StorageReference
 
-const val PHONE_NUMBER = "1234567890"
-
-// Fragment responsible for displaying a list of images from Firebase Storage
 class DisplayFragment : Fragment() {
 
-    // View model for this fragment
-    private val viewModel: DisplayViewModel by viewModels { DisplayViewModelFactory(userID) }
+    private val clientViewModel: ClientDisplayViewModel by viewModels { ClientDisplayViewModelFactory(userFolder) }
+    private val bossViewModel: BossDisplayViewModel by viewModels()
     private val userViewModel: UserViewModel by activityViewModels()
     private var _binding: FragmentDisplayBinding? = null
     private val binding get() = _binding!!
     private val userID = MutableLiveData("Unknown")
-
+    private val userRole = MutableLiveData(Role.UNKNOWN)
+    private val userFolder = MutableLiveData("Unknown")
+    private val PHONE_NUMBER = "1234567890"
```

Comment:
Why PHONE_NUMBER has been removed from the constant ?

Set it as a `private const val` above the class or at least change it from capital letters to camelcase.

Reply:
If you put it outside, the nae of the file in Android Studio will end with a .kt, and it's not uniformed x) And since that will be replaced sometime, I really didn't cared that much, will just put in an companion object for the moment